Userland/Libraries/LibAudio/FlacLoader.cpp

```diff
@@ -417,9 +417,9 @@ u32 FlacLoaderPlugin::convert_sample_count_code(u8 sample_count_code)
         return FLAC_BLOCKSIZE_AT_END_OF_HEADER_16;
     }
     if (sample_count_code >= 2 && sample_count_code <= 5) {
-        return 576 * pow(2, (sample_count_code - 2));
+        return 576 * AK::exp2(sample_count_code - 2);
     }
-    return 256 * pow(2, (sample_count_code - 8));
+    return 256 * AK::exp2(sample_count_code - 8);
```

Comment:
this is doing integer math and I have the feeling this might go negative and break, is that supposed to do that or is it supposed to the divide by the corresponding amount?
@kleinesfilmroellchen